### ISSUE TITLE
Update the my stuff icons

### ISSUE THREAD

**Issue by kccuber-scratch** [ScratchAddons/ScratchAddons#6636]
### Why this would be helpful

original suggestion by jazza

In the my stuff projects list, the icons next to the project that show views, loves, faves, and other aspects about the project are using the old Scratch 2.0 style and are not the new vector .svg icons used in the project player.

### My suggestion

The icons on the my stuff page should be changed to the new scratch 3 icons used in the project player.

### Additional context

This is what the old icons that are on the my stuff page look like:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/92891642/812258c7-7034-4632-bde6-5ab9aef27c20)
These are the new icons which are below the project player:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/92891642/3571b818-12a5-4440-837d-eed3838104af)
thanks to jazza for screenshots too

**Comment by mybearworld** [comment]
I think it would make more sense to put this into `scratchr2` rather than as a new addon.

**Comment by mxmou** [comment]
> I think it would make more sense to put this into `scratchr2` rather than as a new addon.

That's probably what the OP meant too.

**Comment by medlans** [comment]
If this is implemented as a new add on, would there be a way to swap to 2.0 icons and the original icons there (I have a userstyle that brings the 2.0 ones back)? Like you could choose one of the icons and if you want it to show on my stuff, or if you want the vanilla icon.

**Comment by mybearworld** [comment]
> If this is implemented as a new add on, would there be a way to swap to 2.0 icons (if anyone other than me wants it back) and the original 3.0 icons there (I have a userstyle that brings the 2.0 ones back)? Like you could choose one of the icons and if you want it to show on my stuff, or if you want the vanilla icon.

Well, you could just not enable the addon (or `scratchr2` for that matter).

**Comment by MaterArc** [comment]
<img width="892" alt="Screenshot 2023-09-10 at 8 31 07 AM" src="https://github.com/ScratchAddons/ScratchAddons/assets/105017592/e433daf6-85a8-4f27-a08f-c2dbdddba617">

@kccuber-scratch It looks like you are referring to the regular icons and not the colorblind friendly ones, is this intended?

**Comment by mxmou** [comment]
> <img alt="Screenshot 2023-09-10 at 8 31 07 AM" width="892" src="https://user-images.githubusercontent.com/105017592/266835920-e433daf6-85a8-4f27-a08f-c2dbdddba617.png">
> @kccuber-scratch It looks like you are referring to the regular icons and not the colorblind friendly ones, is this intended?

I think it makes sense - the new outlined icons were made to make the difference between selected and unselected states more visible. Icons on the My Stuff page aren't interactive so that isn't necessary.

**Comment by mybearworld** [comment]
> I think it makes sense - the new outlined icons were made to make the difference between selected and unselected states more visible. Icons on the My Stuff page aren't interactive so that isn't necessary.

Well, the colorblind ones are probably better for contrast - also, wouldn't consistency be good here?

**Comment by medlans** [comment]
> > If this is implemented as a new add on, would there be a way to swap to 2.0 icons (if anyone other than me wants it back) and the original 3.0 icons there (I have a userstyle that brings the 2.0 ones back)? Like you could choose one of the icons and if you want it to show on my stuff, or if you want the vanilla icon.
> 
> Well, you could just not enable the addon (or `scratchr2` for that matter).

Well I meant that you would be able to choose your own. That wouldn't allow you to choose the 2.0 ones or the 3.0 icons from before the colorblind changes because there's no setting to do so.

**Comment by BroJac5246** [comment]
> > > If this is implemented as a new add on, would there be a way to swap to 2.0 icons (if anyone other than me wants it back) and the original 3.0 icons there (I have a userstyle that brings the 2.0 ones back)? Like you could choose one of the icons and if you want it to show on my stuff, or if you want the vanilla icon.
> > 
> > Well, you could just not enable the addon (or `scratchr2` for that matter).
> 
> Well I meant that you would be able to choose your own. That wouldn't allow you to choose the 2.0 ones or the 3.0 icons from before the colorblind changes because there's no setting to do so.

mybearworld is saying (and I agree) that if you enable a scratch2r -> scratch-www addon, then it makes sense to have everything be scratch-www. If someone doesn't want scratch-www, then they can choose to not enable the addon.

The other reason not to include a setting is that if this is added then every change could be argued to deserve a setting, which doesn't make sense.

**Comment by mxmou** [comment]
> > I think it makes sense - the new outlined icons were made to make the difference between selected and unselected states more visible. Icons on the My Stuff page aren't interactive so that isn't necessary.
> 
> Well, the colorblind ones are probably better for contrast - also, wouldn't consistency be good here?

The old ones have the same contrast if displayed at 100% opacity. Consistency between different icon in the same area is also important.

**Comment by medlans** [comment]
> > > > If this is implemented as a new add on, would there be a way to swap to 2.0 icons (if anyone other than me wants it back) and the original 3.0 icons there (I have a userstyle that brings the 2.0 ones back)? Like you could choose one of the icons and if you want it to show on my stuff, or if you want the vanilla icon.
> > > 
> > > 
> > > Well, you could just not enable the addon (or `scratchr2` for that matter).
> > 
> > 
> > Well I meant that you would be able to choose your own. That wouldn't allow you to choose the 2.0 ones or the 3.0 icons from before the colorblind changes because there's no setting to do so.
> 
> mybearworld is saying (and I agree) that if you enable a scratch2r -> scratch-www addon, then it makes sense to have everything be scratch-www. If someone doesn't want scratch-www, then they can choose to not enable the addon.
> 
> The other reason not to include a setting is that if this is added then every change could be argued to deserve a setting, which doesn't make sense.

But I said that it didn't effect the project page at all?
Also, it doesn't have to be exactly like I said, it could be made it so when you have scratchr2 -> scratch-www but this on at the same time then it uses the original 3.0 ones or something, I don't know.

**Comment by BroJac5246** [comment]
> > > I think it makes sense - the new outlined icons were made to make the difference between selected and unselected states more visible. Icons on the My Stuff page aren't interactive so that isn't necessary.
> > 
> > Well, the colorblind ones are probably better for contrast - also, wouldn't consistency be good here?
> 
> The old ones have the same contrast if displayed at 100% opacity. Consistency between different icon in the same area is also important.

Could we set the color of the colorblind ones to guarantee contrast?

**Comment by medlans** [comment]
Well, would icons that are fully filled in with a dark color (light if the background is dark) work?? That’s what I was trying to suggest (since that’s the icons that 2.0 used, the background was grey instead though but I think it could work now that it’s white). I was trying to say that I wanted those icons on the project page, and

**Comment by mxmou** [comment]
> Well, would icons that are fully filled in with a dark color (light if the background is dark) work??

Yes, that's what I meant. Customizing the icons on the project page would probably be a new addon though.

**Comment by medlans** [comment]
> > Well, would icons that are fully filled in with a dark color (light if the background is dark) work??
> 
> Yes, that's what I meant. Customizing the icons on the project page would probably be a new addon though.

Same here, I tried to state that you could choose this, but because these happened to be the icons used during 2.0 I referred to them as 2.0 icons, so I think it was interpreted as the My Stuff icons used right now in vanilla Scratch (I thought when I said that I had to get a userstyle for it that people would think I meant ). I was wondering if it would work.
I have a 


> > Well, would icons that are fully filled in with a dark color (light if the background is dark) work??
> 
> Yes, that's what I meant. Customizing the icons on the project page would probably be a new addon though.

Yea I meant that earlier too when I mentioned the new addon thing, but I think people who replied thought I meant the my stuff icons, not the dark ones that used to be used on the 2.0 project page.

**Comment by BroJac5246** [comment]
> > > Well, would icons that are fully filled in with a dark color (light if the background is dark) work??
> > 
> > Yes, that's what I meant. Customizing the icons on the project page would probably be a new addon though.
> 
> Yea I meant that earlier too when I mentioned the new addon thing, but I think people who replied thought I meant the my stuff icons, not the dark ones that used to be used on the 2.0 project page. What I said before was meant to be an alternative to a new addon I guess.

I am very confused. Can you start from the beginning and explain what this is again?

**Comment by DNin01** [comment]
Is [this](https://github.com/ScratchAddons/ScratchAddons/issues/5754) what you're looking for, @medlans?

**Comment by WorldLanguages** [comment]
> Customizing the icons on the project page would probably be a new addon though.

I think we may do that in #6646

**Comment by medlans** [comment]
> Is [this](https://github.com/ScratchAddons/ScratchAddons/issues/5966) the conversation you're looking for, @medlans?


Kinda, but with 2.0 like icons as well or instead of that (I mean these dark ones:

https://en.scratch-wiki.info/wiki/File:Project_stats_2.0.png
I thought these would work well against white, and if the background was dark, then it could become white.
